phase1/gce/gce.jsonnet

```diff
@@ -15,6 +15,7 @@ function(cfg)
     node_firewall_rule: "%(cluster_name)s-node-all" % p1,
     release_bucket: "%s-kube-deploy-%s" % [gce.project, p1.cluster_name],
   };
+  local separate_fields(keyvalues) = std.join(" ", [std.join(" ", std.split(x,"=")) for x in std.split(keyvalues,",")]);
```

Comment:
this function translate a parameter in format "key=value,key=value,.." into a set of values separated for spaces : "key value key value ..." which is a suitable format to be read as an array by bash script.